app/controllers/admin/logs_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Admin
+  class LogsController < Admin::BaseController
+    before_action :retrieve
+
+    def index; end
+
+    def show
+      @path = params[:path]
+      @result = []
+      return unless @files.include?(@path)
+
+      File.open(@path) do |file|
+        file.read.split("\n").each do |line|
+          @result.push line
+        end
+      end
+    end
+
+    private
+
+      def retrieve
+        path = Rails.root.join('log/*')
```

Comment:
__[RuboCop]__

Rails/FilePath: Please use `Rails.root.join('path', 'to')` instead.


<!-- Comment by [SideCI](https://sideci.com) -->